spatio_flux/processes/dfba.py

```diff
@@ -105,57 +114,85 @@ def update(self, state, interval):
         else:
             # Handle non-optimal solutions if necessary
             # print('Non-optimal solution, skipping update')
-            for substrate, reaction_id in self.config['substrate_update_reactions'].items():
+            for substrate, reaction_id in self.config[
+                "substrate_update_reactions"
+            ].items():
                 substrate_update[substrate] = 0
 
         return {
-            'substrates': substrate_update,
+            "substrates": substrate_update,
         }
 
 
 # register the process
-core.register_process('DynamicFBA', DynamicFBA)
+core.register_process("DynamicFBA", DynamicFBA)
 
 
 # Helper functions to get specs and states
 def dfba_config(
-        model_file='textbook',
-        kinetic_params=None,
-        biomass_reaction='Biomass_Ecoli_core',
-        substrate_update_reactions=None,
-        biomass_identifier='biomass',
-        bounds=None
+    model_file="textbook",
+    kinetic_params=None,
+    biomass_reaction="Biomass_Ecoli_core",
+    substrate_update_reactions=None,
+    biomass_identifier="biomass",  # TODO: How do we differentiate between biomass between models? Do we need to change the biomass identifier to be unique for each model?
+    bounds=None,
 ):
+    if kinetic_params is None:
+        kinetic_params = {"glucose": (0.5, 1), "acetate": (0.5, 2)}
+    # TODO: Look for the biomass reaction in the model, so it doesn't have to be specified
+    # if biomass_reaction is None:
+    # biomass_reaction = get_biomass_reaction(model_file)
```

Comment:
Good idea. There's plenty of room for more automated extraction of relevant information with different COBRA functions.